CHANGELOG.md

```diff
@@ -1,25 +1,58 @@
 # Change Log
 
-## 0.6.0 - 2023-10-30
+## 0.6.0 - 2023-10-31
 
 ### Added
-* Revision management tree node for multiple revision mode [#379](https://github.com/microsoft/vscode-azurecontainerapps/pull/379)
-* `Deploy Workspace to Container App...` command by @MicroFish91 and @motm32 see issue [#464](https://github.com/microsoft/vscode-azurecontainerapps/issues/425)
-* Revision draft support by @MicroFish91 in [#405](https://github.com/microsoft/vscode-azurecontainerapps/pull/405),[#413](https://github.com/microsoft/vscode-azurecontainerapps/pull/413), [#414](https://github.com/microsoft/vscode-azurecontainerapps/pull/414)
-* Support for deploying to a container app via a connected Github reository and enable CI by @MicroFish91 in [#353](https://github.com/microsoft/vscode-azurecontainerapps/pull/353), [#355](https://github.com/microsoft/vscode-azurecontainerapps/pull/355), [#361](https://github.com/microsoft/vscode-azurecontainerapps/pull/361), [#363](https://github.com/microsoft/vscode-azurecontainerapps/pull/361), [#407](https://github.com/microsoft/vscode-azurecontainerapps/pull/407)
-* Log streaming support by @motm32 in [#350](https://github.com/microsoft/vscode-azurecontainerapps/pull/350)
-* Support for editing secrets by @MicroFish91 in [#419](https://github.com/microsoft/vscode-azurecontainerapps/pull/419)
+* Large rework of the container apps UI
+    * Added a new `Configurations` item to the container app item that is present in all views.  The `Configurations` item houses the old `Dapr` and `Ingress` tree items as well as the new GitHub `Actions` and `Secrets` tree items [#379](https://github.com/microsoft/vscode-azurecontainerapps/pull/379)
```

Comment:
Could be good to be consistent and call all the tree items either items or tree items.